hc-sr04-range-sensor/Makefile

```diff
@@ -92,6 +100,70 @@ $(ELF1): $(SRC1) $(HEADERS) | $(OUT)
 clean:
 	$(RM) -fr $(ELF0) $(ELF1) $(OUT)
 
+install_PRU0:
+	@echo 'Installation Process Initiated'
+ifeq ($(CURRENTSTATE0),$(STATESRUNNING))
+	@echo 'Stopping current PRU0 application ($(PRU0))'
+	@echo stop | sudo tee $(PRU0)/state
+endif
+	@echo 'Installing firmware'
+	@sudo cp $(ELF0) /lib/firmware/am335x-pru0-fw
+	@echo 'Starting new PRU0 application'
+	@echo start | sudo tee $(PRU0)/state
+
+install_PRU1:
+	@echo ' Installation Process Initiated'
+ifeq ($(CURRENTSTATE1),$(STATESRUNNING))
+	@echo 'Stopping current PRU1 application ($(PRU1))'
+	@echo stop | sudo tee $(PRU1)/state
+endif
+	@echo 'Installing firmware'
+	@sudo cp $(ELF1) /lib/firmware/am335x-pru1-fw
+	@echo 'Starting new PRU1 application'
+	@echo start | sudo tee $(PRU1)/state
+
+stop_PRU0:
+	@echo 'Process Initiated'
+ifeq ($(CURRENTSTATE0),$(STATESRUNNING))
+	@echo 'Stopping current PRU0 application ($(PRU0))'
+	@echo stop | sudo tee $(PRU0)/state
+	@echo 'Stopped PRU0'
+else
+	@echo 'PRU0 Already in the stopped state. execute start_PRU0 to start '
+endif
+
+stop_PRU1:
+	@echo 'Process Initiated'
+ifeq ($(CURRENTSTATE1),$(STATESRUNNING))
+	@echo 'Stopping current PRU0 application ($(PRU1))'
+	@echo stop | sudo tee $(PRU1)/state
+	@echo 'Stopped PRU1'
+else
+	@echo 'PRU0 Already in the stopped state. execute start_PRU0 to start '
+endif
+
+start_PRU0:
```

Comment:
Please adhere to the existing convention in the Makefile all rules to be only lower case.